integrations/influxdb_connector/influxdb-timestream-connector/src/lib.rs

```diff
@@ -67,13 +67,20 @@ async fn handle_ingestion(
     let database_name = std::env::var("database_name")?;
     let database_name = Arc::new(database_name);
 
+    let kms_key_id = std::env::var("kms_key_id").ok();
+
+    let tags = match std::env::var("tags") {
+        Ok(tags_json) => Some(parse_tags_from_json(&tags_json)?),
```

Comment:
Have you considered an alternative format to JSON? With the supported tag characters, maybe CSV would be a preferred format for readability.